packages/babel-helper-create-class-features-plugin/src/fields.ts

```diff
@@ -510,16 +510,34 @@ function buildPrivateInstanceFieldInitSpec(
   ref: t.Expression,
   prop: NodePath<t.ClassPrivateProperty>,
   privateNamesMap: PrivateNamesMap,
+  state,
 ) {
   const { id } = privateNamesMap.get(prop.node.key.id.name);
   const value = prop.node.value || prop.scope.buildUndefinedNode();
 
-  return template.statement.ast`${t.cloneNode(id)}.set(${ref}, {
-    // configurable is always false for private elements
-    // enumerable is always false for private elements
-    writable: true,
-    value: ${value},
-  })`;
+  if (
+    !process.env.BABEL_8_BREAKING &&
+    !state.availableHelper("classPrivateFieldInitSpec")
+  ) {
+    return template.statement.ast`${t.cloneNode(id)}.set(${ref}, {
+      // configurable is always false for private elements
+      // enumerable is always false for private elements
+      writable: true,
+      value: ${value},
+    })`;
+  }
```

Comment:
This needs to be two separate `if`s. Our logic to conditionally remove Babel 7 code or Babel 8 code at release time isn't smart enough :grimacing: 

```js
if (!process.env.BABEL_8_BREAKING) {
  if (!state.availableHelper("classPrivateFieldInitSpec")) {
    ...
  }
}
```